pddlgym/spaces.py

```diff
@@ -89,6 +89,25 @@ def _compute_all_ground_literals(self, state):
                 all_ground_literals.add(lit)
         return all_ground_literals
 
+    def contains(self, x):
+        """Return boolean if x is a valid member of this space."""
+        if not isinstance(x, State):
+            return False
+
+        # Check all predicates can be grounded from the objects in the state.
+        try:
+            self._update_objects_from_state(x)
+        except:
```

Comment:
can we catch a more specific type of exception here? feel free to change `_update_objects_from_state` to raise a specific type of error if needed. naked `except` statements worry me because they can lead to silent bugs

Reply:
looks great